tests/test_acasclient.py

```diff
@@ -2361,6 +2361,8 @@ def get_basic_experiment_load_file_with_project(project_code, tempdir, corp_name
     # Read the data file and replace the project code with the one we want
     with open(data_file_to_upload, 'r') as f:
         data_file_contents = f.read()
+    f.close()
```

Comment:
These shouldn't be necessary. The point of the `with` block is that it calls `close` https://www.geeksforgeeks.org/with-statement-in-python/

Reply:
Ah sorry, didn't mean to make any of that part of this PR, was trying to fix a bunch of warnings I saw on the tests.

Going to remove those parts of the code not related to the dry run parameters passing.